joomla-extensions/install-from-web-server
# Stage one: CleanUp com_apps Administrator

### What this PR does:

- apply code style to the most files
- move the language folder
- remove index.html
- update copyright
- give the admin page some love:

![com_apps_new](https://cloud.githubusercontent.com/assets/2596554/6337148/22c2218c-bba8-11e4-96e5-7349afd00565.PNG)

### How to test
- apply patch
- see com_apps admin view
- compare to the screenshot above
